src/test/kotlin/lotto/domain/MoneyTest.kt

```diff
@@ -0,0 +1,24 @@
+package lotto.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class MoneyTest {
+    @Test
+    fun `머니 객체가 음수를 받았을 때 예외 발생 테스트`() {
+        assertThrows<IllegalArgumentException> {
+            Money(-1)
+        }
+    }
+
+    @Test
+    fun `머니 객체의 더하기연산 테스트`() {
+        val sumResultMoney = Money(3) + Money(5)
+
+        val answer = sumResultMoney.money
+        val expect = 8
+
+        assertThat(answer).isEqualTo(expect)
```

Comment:
value class(data class)의 특징을 이해한다면, 아래와 같이 작성할 수 있을 것 같아요!
[참고](https://kotlinlang.org/docs/data-classes.html)

```suggestion
        assertThat(sumResultMoney).isEqualTo(Money(8))
```